src/exceptions/LLVM-exception.xml

```diff
@@ -5,6 +5,9 @@
          <crossRef>http://lists.llvm.org/pipermail/llvm-dev/2017-August/116266.html</crossRef>
       </crossRefs>
       <notes>This exception was created specifically to be used with Apache-2.0</notes>
+	  <titleText>---- LLVM Exceptions to the Apache 2.0 License ----
```

Comment:
I think these dashes are part of header markup.  I'd rather see them optional inside content-less `<alt>` tags (see 8b32737, #515, for a similar change to a different license.